Tests/Marketplace/copy_and_upload_packs.py

```diff
@@ -143,6 +143,7 @@ def upload_core_packs_config(production_bucket: Bucket, build_number: str, extra
     # construct core pack data with public gcs urls
     core_packs_data = {
         'corePacks': corepacks_list,
+        'upgradeCorePacks': corepacks_file.get('upgradeCorePacks', []),
```

Comment:
Please add a suitable docstring for this function regarding the upgradeCorePacks part

Reply:
Done